### PR TITLE
Edb refactor

### PR DESCRIPTION
**Pull request by maxcapodi78**

### CODE REVIEW

**Review by svandenb-dev — APPROVED** [review]

File: _unittest/test_26_emit.py
Comment (myoung301) [inline _unittest/test_26_emit.py:1]: Why is this file changing? It looks like the changes shown here are already on main, but if that's the case, I'm not sure why I'm seeing them in this PR.

File: pyaedt/emit_core/results/revision.py
Comment (myoung301) [inline pyaedt/emit_core/results/revision.py:1]: I'm seeing the same thing in this file - new sections are shown in the diff for this PR that are already on main. Am I looking at something wrong?

**Review by myoung301 — CHANGES_REQUESTED** [review]
It looks like there are extra changes in the PR when I review the diffs. Is there a problem with the PR or am I looking at something wrong?

File: _unittest/test_26_emit.py
Comment (maxcapodi78) [reply _unittest/test_26_emit.py:1]: because I've merged from main

File: pyaedt/emit_core/results/revision.py
Comment (maxcapodi78) [reply pyaedt/emit_core/results/revision.py:1]: you see because I've merged from main

File: _unittest/test_26_emit.py
Comment (myoung301) [reply _unittest/test_26_emit.py:1]: Ok - I see it's a 'limitation' of GitHub. Looks like there are some ways to get a true diff though. 

https://stackoverflow.com/questions/16306012/github-pull-request-showing-commits-that-are-already-in-target-branch

This did the trick:

https://github.com/ansys/pyaedt/compare/main...edb_refactor

**Review by myoung301 — APPROVED** [review]

**Pull request merged**